### ISSUE TITLE
Sound design question: spatial audio for position sliders?

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#126]
@Ashton-Morris and @jbphet, due to the lack of focus and blur events available on mobile platforms, we can't use some of the position-orientation phrases as I designed them. These phrases are meant to provide users with some indication of where the mass sphere (that they are currently focused on) is a long the track. Specifically these phrases: "Left side of track", "Right side of track, "Last stop right", "Last stop left", "Closer", and "Farther away", cannot depend on focus or blur events. 

We are in the process of re-designing how/when these phrases are provided (see #124), and in addition @emily-phet suggested I reach out to the both of you to see how easy it would be to add positional audio to the "Move mass 1"/"Move mass 2" sliders?

@emily-phet and I are wondering how much effort is needed to do this. Is it easy-peasy, really difficult, or some where in between?

We do not want to make a change at this point that would significantly change the publication date of this sim.

**Comment by jbphet** [comment]
It's somewhere in between easy and really difficult.  Web Audio has good support for panning, and I did some experimentation with it early on, but we largely decided not to do much with it because we couldn't always count on users wearing headphones, and the stereo imaging is a bit too subtle to rely on when coming from laptop speakers.  Because of this, tambo (the sound library) doesn't currently have any support for panning sounds to the the left or right.  I'd estimate that it would take me two work days to add and test it.

**Comment by jbphet** [comment]
I just realized that I made a bit of an assumption in my previous comment.  I assumed that "positional audio" means left and right panning, and this thinking was influenced by the use of the term 'spatial audio' in the title.  If that sort of panning of the audio is not what is being discussed, please let me know.

**Comment by Ashton-Morris** [comment]
I had previously asked John if he thought panning the sounds would help certain users. But we weren't sure how noticeable it would be to most users or what listening environment they would be in. 

That type of work would be main on Johns end though if you are talking about L R panning.

**Comment by terracoda** [comment]
@jbphet and @Ashton-Morris, yes, I think I am talking about L R panning. I agree that it would not be available in all situations, but I wanted to reach out to see if it was hard or easy to see if it was worth doing to compensate for some environments.

Thanks for your comments. I'll discuss with @emily-phet.

**Comment by terracoda** [comment]
We'll put it on the wish list for Tambo, but no worry about it right now.
